web_src/less/_base.less

```
@@ -230,10 +233,10 @@ progress::-webkit-progress-bar {
   background: var(--color-secondary-dark-1);
 }
 progress::-webkit-progress-value {
-  background-color: var(--color-secondary-dark-4);
+  background-color: var(--color-primary);
```

Shouldn't this be
```suggestion
  background-color: var(--color-accent);
```
?

---

Ideally, yes.